src/main/java/uk/gov/ida/matchingserviceadapter/configuration/EuropeanIdentityConfiguration.java

```diff
@@ -33,8 +32,8 @@
     @JsonProperty
     private EidasMetadataConfiguration aggregatedMetadata;
 
-    public String getHubConnectorEntityId() {
-        return hubConnectorEntityId;
+    public String getHubConnectorEntityId(MatchingServiceAdapterEnvironment environment) {
+        return Optional.ofNullable(hubConnectorEntityId).orElse(getAcceptableHubConnectorEntityIds(environment).stream().findFirst().orElse(null));
```

Comment:
I don't think that's right. If `getHubConnectorEntityId ` is not present then we should return `Optional.empty` (change type to `Optional<String>`)

Reply:
👍 I have removed this now - we don't use `getHubConnectorEntityId()`